index.js

```diff
@@ -0,0 +1,4 @@
+const server = require('./api/server.js');
+
+const port = process.env.PORT || 6000;
+server.listen(port, () => console.log('\n** api up on port 6k **\n'));
```

Comment:
Since you're using that process.env.PORT, in theory, the port might be something other than 6000, so you should use the port variable in your console.log.